resources/monitoring/charts/grafana/README.md

```diff
@@ -51,6 +51,21 @@ Users can create their own **Grafana Dashboard** by using the Grafana UI as the
 1. Create or Modify a dashboard using Grafana UI

```

Comment:
Could you also change this one to:
`Create or modify a dashboard using Grafana UI.` ? 

It's not changed within this task, but for consistency we can change it :)